accord-core/src/test/java/accord/impl/basic/LoggingJournal.java

```diff
@@ -73,6 +74,13 @@ private synchronized void log(String format, Object... objects)
         }
     }
 
+    @Override
+
+    public Journal start(Node node)
+    {
+        return this;
+    }
```

Comment:
```suggestion
    @Override
    public Journal start(Node node)
    {
        delegate.start(node);
        return this;
    }
```

Reply:
I’ll have to look at this again next week. I think @ifesdjeen may have said this startup sequencing issue (I forgot the precise problem) may have been addressed in another patch of his anyway.